### DISCUSSION TITLE
i can't install packages or themes

### DISCUSSION THREAD

**Discussion by vulpesIo** [vercel/hyper#7292] [Q&A] [answered]
i tried to install a theme .
`hyper i hypercwd`

but when i do i get this error im new to this an i can't find any documentation on this.
`bash: hyper: command not found`

I would apreciate

**Comment by roma-marshall** [comment, marked as answer]
hello @fox5352, check it out: [#3349](https://github.com/vercel/hyper/issues/3349)

**Reply by vulpesIo** [reply]
Thanks for the help
